pkg/systemd/service-details.jsx

```diff
@@ -421,100 +421,154 @@ export class ServiceDetails extends React.Component {
         const isTimer = (this.unitType === "timer");
         const isQuadlet = this.props.unit.SourcePath?.includes("/containers/systemd/");
 
-        let status = [];
-
-        if (masked) {
-            status.push(
-                <div key="masked" className="status-masked">
-                    <BanIcon className="status-icon" />
+        let statuses = [];
+
+        if (masked || true) {
+            statuses.push(
+                <ListItem
+                    key="masked"
+                    className="status-masked"
+                    icon={
+                        <Icon>
+                            <BanIcon className="status-icon" />
+                        </Icon>
+                    }
+                >
                     <span className="status">{ _("Masked") }</span>
                     <span className="side-note font-xs">{ _("Forbidden from running") }</span>
-                </div>
+                </ListItem>
             );
         }
 
         if (!enabled && !active && !masked && !isStatic) {
-            status.push(
-                <div key="disabled" className="status-disabled">
-                    <OffIcon className="status-icon" />
-                    <span className="status">{ _("Disabled") }</span>
-                </div>
+            statuses.push(
+                <ListItem
+                    key="disabled"
+                    className="status-disabled"
+                    icon={
+                        <Icon>
+                            <OffIcon className="status-icon" />
+                        </Icon>
+                    }
+                >
+                    <span className="status">{_("Disabled")}</span>
+                </ListItem>
             );
         }
 
         if (failed) {
-            status.push(
-                <div key="failed" className="status-failed">
-                    <ErrorCircleOIcon className="status-icon" />
+            statuses.push(
+                <ListItem
+                    key="failed"
+                    className="status-failed"
+                    icon={
+                        <Icon>
+                            <ErrorCircleOIcon className="status-icon" />
+                        </Icon>
+                    }
+                >
                     <span className="status">{ _("Failed to start") }</span>
                     { showAction &&
-                    <Button variant="secondary" className="action-button" onClick={() => this.unitAction("StartUnit") }>{ _("Start service") }</Button>
+                        <Button variant="secondary" className="action-button" onClick={() => this.unitAction("StartUnit")}>
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22251-16fefbfb-20250725-220859-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)